userguide/plugins.rst

```diff
@@ -192,23 +192,24 @@ To update or upgrade the plugin jail operating system, see
 
 .. _Deleting Plugins:
 
-Delete
-------
+Deleting Plugins
+----------------
 
 Installing a plugin creates an associated jail. Deleting a plugin
-deletes the associated jail because it is no longer required.
-**Before** deleting a plugin, make sure that there is no data
-or configuration in the jail that needs to be saved. Back up
-that data **first** if needed.
+deletes the associated jail because it is no longer required. This
+means all **datasets or snapshots that are associated with the plugin
+are also deleted.** Make sure to back up any important data from the
+plugin **before** deleting it.
 
-In the example shown in
-:numref:`Figure %s <deleting_installed_plugin_fig>`,
-*plex* has been installed and the :guilabel:`UNINSTALL` button has
-been clicked. A pop-up message asks for verification that the plugin
-is to be deleted. **This is the only warning.** The plugin and the
-associated jail are permanently deleted when :guilabel:`Confirm` is
-set and :guilabel:`DELETE` is clicked.
+The plugin jail must be *down* before it can be uninstalled. Expand the
```

Comment:
> Expand the plugin entry and click :guilabel:`STOP` to shut down the plugin.

Suggestion:
``Click |ui-chevron-right| to expand the plugin. Click :guilabel:`STOP` to shut down the plugin. ``